.gitignore

```diff
@@ -15,3 +15,5 @@ coverage.xml
 nosetests.xml
 junit-*.xml
 build
+.idea/
```

Comment:
Can we actually change this to GitHubs Python gitignore template? Covers a lot more bases! Then add `test_elasticsearch/local.py` and `.idea` to it.